core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -178,6 +178,8 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_MAT_VIEW_INSERT_AS_SELECT_BATCH_SIZE("cairo.mat.view.insert.as.select.batch.size"),
     CAIRO_MAT_VIEW_ROWS_PER_QUERY_ESTIMATE("cairo.mat.view.rows.per.query.estimate"),
     CAIRO_MAT_VIEW_PARALLEL_SQL_ENABLED("cairo.mat.view.parallel.sql.enabled"),
+    CAIRO_MAT_VIEW_MAX_REFRESH_INTERVALS("cairo.mat.view.max.refresh.intervals"),
+    CAIRO_MAT_VIEW_REFRESH_INTERVALS_UPDATE_INTERVAL("cairo.mat.view.refresh.intervals.update.interval"),
```

Comment:
nit: 
```suggestion
    CAIRO_MAT_VIEW_REFRESH_INTERVALS_UPDATE_INTERVAL("cairo.mat.view.refresh.intervals.update.period"),
```

Reply:
WAL purge job's prop is called `cairo.wal.purge.interval`. We have some more examples like `cairo.wal.sequencer.check.interval` or `cairo.idle.check.interval`.

Reply:
Two uses of the word `period` make the name confusing. But it's fine, if you don't like it, you can keep this name

Reply:
Renamed in c25eb15c130db6e817f1797f099fa297ac28f06a